crates/hir_def/src/item_tree.rs

```diff
@@ -104,6 +104,11 @@ impl ItemTree {
                     // items and expanded during block DefMap computation
                     return Default::default();
                 },
+                ast::Type(_ty) => {
+                    // FIXME: This occurs because macros in type position are treated as inner
+                    // items and expanded during block DefMap computation
+                    return Default::default();
```

Comment:
Shouldn't this mirror the case below, since types can contain inner items?

Reply:
> since types can contain inner items?

(I am curious, I can't think of an example where a type has inner items?)

Reply:
`[u8; { const fn len() { 7 } len() }]`